### PR TITLE
SAK-31460 Changes for optional Quick Links functionality.

### PR DESCRIPTION
**Pull request by RebeccaOxfordUni**

### CODE REVIEW

**Comment by buckett** [comment]
Build fails and looks related to this change.

**Comment by RebeccaOxfordUni** [comment]
Minor change made.

File: reference/library/src/morpheus-master/sass/tool.scss [hunk 1]
```diff
@@ -44,6 +44,7 @@
 @import "base/rtl";
 
 @import "modules/access/base";
+@import "modules/quicklinks/_base";
```
Comment (buckett) [inline reference/library/src/morpheus-master/sass/tool.scss]: I think this should be `/base` to keep it the same as the surrounding lines.

**Comment by jonespm** [comment]
Added some comments to the jira

**Comment by ottenhoff** [comment]
Should we keep this open?

**Comment by buckett** [comment]
I think @RebeccaOxfordUni is going to look at sorting this out.

**Comment by RebeccaOxfordUni** [comment]
I've added some comments in the Jira.

**Comment by RebeccaOxfordUni** [comment]
Hopefully fixed the issue that was breaking the build.

**Comment by jonespm** [comment]
It looks like this is missing some commits.
1) I get an error in the UI missing key (mre): sitenav sit_quick_links]
  - I don't see any message bundle additions committed here
2) Why is it called 'related links' in the code rather than 'quick links' this should be consistent
3) Property should be prefixed with portal. and also probably called quicklink too rather than related.link?
  - portal.quicklink.url=
4) I'd remove the oxford from the property example since the image isn't there and the example doesn't work. Also using new style propreties is better (This would be portal.quicklink.url)
```
related.link.url=https://sakaiproject.org/,https://www.apereo.org
related.link.title=Sakai Project Website,Apereo Website
related.link.name=Sakai Project,Apereo
related.link.icon=im /library/image/sakai/url.gif,cl fa fa-glob
```

**Comment by jonespm** [comment]
It also doesn't seem to be hiding the relatedLinks div to me? Maybe this is just some mixup with some parts being quickLinks and some being relatedLinks? 

image

File: portal/portal-render-engine-impl/pack/src/webapp/vm/morpheus/includeLoginNav.vm [hunk 2]
```diff
@@ -3,11 +3,51 @@
 <nav id="mastLogin" class="Mrphs-loginNav">
 
     #if (!${loginTopLogin})
+        <div id="relatedLinks">
```
Comment (jonespm) [inline portal/portal-render-engine-impl/pack/src/webapp/vm/morpheus/includeLoginNav.vm]: This div seems unnecessary with the quickLinks div below?

**Comment by RebeccaOxfordUni** [comment]
Thanks for the feedback Matthew.  I've added the sitenav properties and changed the property names to be the new style you mention above.  The code now calls everything quick links instead of related links.

The extra div you mentioned has been removed which should sort out the quicklinks showing when they shouldn't.  I'll review cherry-picking from our Oxford code more thoroughly next time!

Thanks once again for looking at this.

**Comment by jonespm** [comment]
I tested this out and think it works good, keyboard focus works too which is good for a11y. Thanks for the changes!

**Pull request merged**